### PR TITLE
feat: NFT Pack Opening 

### PR DESCRIPTION
**Pull request by zaxozhu**
**NFT Packs Redeem**
This feature allows the user to view and open NFT packs (any number of NFTs as part of one essence — pack).

In this PR
This PR consists of:
* UI implementation for Pack Redeeming
* A set of transactions to open an NFT pack.
* Updates on My Items page to fetch NFTs each time the user visits a page
* Removal of WebSockets from useMeta

Is hidden behind **REACT_APP_ENABLE_NFT_PACKS** .env flag.

Tickets:
https://app.asana.com/0/1201289799176936/1201399761117040/f
https://app.asana.com/0/1201289799176936/1201399761117045/f
https://app.asana.com/0/1201289799176936/1201399761117043/f
https://app.asana.com/0/1201289799176936/1201364603997898/f
https://app.asana.com/0/1201289799176936/1201399761117037/f

### CODE REVIEW

**Review by ViacheslavMylostyvyi — APPROVED** [review]

**Review by ikadushkina — APPROVED** [review]

File: js/packages/common/src/utils/ids.ts [hunk 1]
```diff
@@ -78,7 +78,11 @@ export const METAPLEX_ID =
   'p1exdMJcjVao65QdewkaZRUnU6VPSXhus9n2GzWfh98' as StringPublicKey;
 
 export const PACK_CREATE_ID = new PublicKey(
-  'packFeFNZzMfD9aVWL7QbGz1WcU7R9zpf6pvNsw2BLu',
+  'GTFVL9hA9AhWjNxhH99Hw2mp1MycFqRTjDS4xAHREheC',
```
Comment (zaxozhu) [inline js/packages/common/src/utils/ids.ts:81]: @austbot We need to deploy the latest codebase to packFeFNZzMfD9aVWL7QbGz1WcU7R9zpf6pvNsw2BLu, so I can change the program id here

File: js/packages/common/src/utils/ids.ts [hunk 2]
```diff
@@ -78,7 +78,11 @@ export const METAPLEX_ID =
...
+);
+
+export const ORACLE_ID = new PublicKey(
+  'HS66XZHFBWQAcXa124E9qkxFB2JQpm9AhXdD4Q4dtLsY',
```
Comment (zaxozhu) [inline js/packages/common/src/utils/ids.ts:85]: @austbot This is the oracle @VadimGrozinok gave to me

File: js/packages/common/src/contexts/meta/loadAccounts.ts [hunk 3]
```diff
@@ -975,6 +1096,47 @@ const pullMetadataByCreators = (
...
+      }),
+    );
+    setOf100MetadataEditionKeys = [];
+  };
+
+  for (const metadata of metadataKeys) {
+    setOf100MetadataEditionKeys.push(metadata);
+
+    if (setOf100MetadataEditionKeys.length >= 100) {
+      loadBatch();
```
[the remark this answers is not in the document]
Comment (msebastiian) [reply js/packages/common/src/contexts/meta/loadAccounts.ts:1128]: It works the same way as in **pullEditions**

setOf100MetadataEditionKeys is being reset on line 1121

[the remark this answers is not in the document]
Comment (zaxozhu) [reply js/packages/common/src/contexts/meta/loadAccounts.ts:1128]: It works the same way as in pullEditions

setOf100MetadataEditionKeys is being reset on line 1121

**Pull request merged**